hamocc/mo_carbch.F90

```diff
@@ -304,18 +305,28 @@ SUBROUTINE ALLOC_MEM_CARBCH(kpie,kpje,kpke)
       if(errstat.ne.0) stop 'not enough memory atmflx'
       atmflx(:,:,:) = 0.0
 
-      ! Allocate field to hold N-deposition fluxes per timestep for inventory caluclations
+      ! Allocate field to hold N-deposition fluxes per timestep for inventory caluclations and output
```

Comment:
Nit picky, but if you change the line, correct the spelling as well :-)    (calculations - same below l. 319)

Reply:
Ok, done!